### PR TITLE
dd style

### PR DESCRIPTION
**Pull request by Dove-white**
# Summary
## Implemented the following:

- Add styling to app

### CODE REVIEW

File: css/style.css [hunk 1]
```diff
@@ -140,13 +140,18 @@
   border-radius: 3px;
 }
 
+.book:hover {
+  cursor: pointer;
+}
```
Comment (SekoViper) [inline css/style.css]: Kindly add the css property in the class itself not on hover.

**Review by SekoViper — CHANGES_REQUESTED** [review]
Hi @Dove-white , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights

- [ ] Correct Gitflow ✔️ 
- [ ] Descriptive PR ✔️ 

## Required Changes ♻️

- [ ] Please kindly check the comment under the review 👇 👇 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

File: css/style.css (hunk 1 above)
Comment (Dove-white) [reply css/style.css]: kk boss

**Review by SekoViper — APPROVED** [review]
# Status: ✔️ Project Approved ✔️ 

Hi @Dove-white ,


Everything is looking great! Please, merge this branch :shipit:
Congratulations! 🎉

## Highlights

 - Clean and organized code. ✔️ 
 - Descriptive commit messages.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**Pull request merged**